secondQuestion.js

```diff
@@ -0,0 +1,18 @@
+function dot_product(v1, v2) {
+    var dotProduct= -1;
+    var orthogonal=false;
```

Comment:
Both `dotProduct` and `orthogonal` variables could be defined using `let`, no need to use `var`.